### PR TITLE
feat: support arm64 client instance

### PR DESCRIPTION
**Pull request by denise89p**

### CODE REVIEW

File: modules/clients/init.sh [hunk 1]
```diff
@@ -52,7 +54,7 @@ while ! ip route | grep eth$i; do
   ip route
   sleep 5
 done
-/usr/sbin/ip route del ${subnet_range} dev eth$i
+ip route del ${subnet_range} dev eth$i
```
[the remark this answers is not in the document]
Comment (denise89p) [reply modules/clients/init.sh:57]: yes

File: modules/clients/main.tf [hunk 2]
```diff
@@ -26,6 +26,7 @@ locals {
 
   custom_data_parts = [local.preparation_script, local.mount_wekafs_script]
   vms_custom_data   = base64encode(join("\n", local.custom_data_parts))
+  use_arch          = var.arch == "arm64" ? 1 : 0
```
Comment (assafgi) [inline modules/clients/main.tf:29]: why do you need it?

File: modules/clients/main.tf [hunk 3]
```diff
@@ -107,7 +108,8 @@ resource "azurerm_linux_virtual_machine" "this" {
   tags                = merge({ "weka_cluster" : var.clients_name })
   custom_data         = local.vms_custom_data
   source_image_id     = var.source_image_id
-  size                = var.instance_type
```
Comment (assafgi) [inline modules/clients/main.tf:110]: why ? previous formatting looks right

File: modules/clients/main.tf (hunk 2 above)
Comment (denise89p) [reply modules/clients/main.tf]: removed

File: modules/clients/main.tf (hunk 3 above)
Comment (denise89p) [reply modules/clients/main.tf]: fixed

**Pull request merged**